cmd/templ/generatecmd/proxy/proxy.go

```diff
@@ -114,7 +166,43 @@ func (h *Handler) modifyResponse(r *http.Response) error {
 	case "":
 		log.Debug("No content encoding header found")
 	default:
-		h.log.Warn(unsupportedContentEncoding, slog.String("encoding", r.Header.Get("Content-Encoding")))
+		log.Warn(unsupportedContentEncoding, slog.String("encoding", r.Header.Get("Content-Encoding")))
```

Comment:
This log line using the unwrapped logger seemed unintentional to me, so I changed it to use the wrapped one